File: PapaKarloSwift/PapaKarloSwift/feature/consumer_cart/CartProductItem.swift
//
//  CartProductItem.swift
//  PapaKarloSwift
//
//  Created by Марк Шавловский on 26.03.2022.
//

import SwiftUI

struct CartProductItem: Identifiable {
    var id :String
    let name:String
    let newCost:String
    let oldCost:Int?
    let photoLink:String
    let count:Int
    let menuProductUuid:String
}

File: app/src/main/java/com/bunbeauty/papakarlo/feature/consumer_cart/model/CartProductItem.kt
package com.bunbeauty.papakarlo.feature.consumer_cart.model

data class CartProductItem(
    val uuid: String,
    val name: String,
    val newCost: String,
    val oldCost: String?,
    val photoLink: String,
    val count: Int,
    val menuProductUuid: String
)
